File: LATIHAN2/SivitasAkademik.java
// kelas sivitas anak dari human
public class SivitasAkademik extends Human {
  // delarasi variabel private untuk dalam kelas
  private String universitas;
  private String email;

  // konstruktor
  public SivitasAkademik(String NIK, String nama, String jk, String universitas,
                         String email) {
    // ambil dari ortunya
    super(NIK, nama, jk);
    this.universitas = universitas;
    this.email = email;
  }
  // prosedur print
  public void display() {
    super.display();
    System.out.println("Universitas   : " + universitas);
    System.out.println("Email         : " + email);
  }
  // getter setter
  public String getUniversitas() { return universitas; }
  public void setUniversitas(String universitas) {
    this.universitas = universitas;
  }

  public String getEmail() { return email; }
  public void setEmail(String email) { this.email = email; }
}

File: LATIHAN2/PHP/SivitasAkademik.php
<?php
require_once 'C:/xampp/htdocs/DPBO/PRAK/LATIHAN2/Human.php';
//anak dari human
class SivitasAkademik extends Human {
  private $universitas;
  private $email;

    //construct
  public function __construct($NIK, $nama, $jk, $universitas, $email) {
    parent::__construct($NIK, $nama, $jk);
    $this->universitas = $universitas;
    $this->email = $email;
  }

    //methode print 
  public function display() {
    parent::display();//print parent
    echo "Universitas   : " . $this->universitas . "<br>";
    echo "Email         : " . $this->email . "<br>";
  }

    //getter setter
  public function getUniversitas() { return $this->universitas; }
  public function setUniversitas($universitas) { $this->universitas = $universitas; }

  public function getEmail() { return $this->email; }
  public function setEmail($email) { $this->email = $email; }
}
